app/styles/main.scss

```diff
@@ -1,8 +1,9 @@
 $color-primary: #2d3e4e;
-$color-secondary: #9e0c0f;
+$color-secondary: #DAA520;
 $color-accent: #f5b90b;
 $color-light-text: #a6a99e;
 $color-dark-text: #7f7f7f;
+$color-standard: #4d4d4d;
```

Comment:
I think we can do two things:

1. Consolidate all default variables in `default-variables`
2. Move UMUC-specific variables to a separate file, similar to how isolated all UMUC-specific CSS rules in separate file

Can you move the following to variables to `default-variables.scss`?:

```
$color-primary: #2d3e4e;
$color-secondary: #9e0c0f;
$color-accent: #f5b90b;
$color-light-text: #a6a99e;
$color-dark-text: #7f7f7f;
$color-standard: #4d4d4d;
```

And then add a file `custom-umuc-variables.scss`, which includes the following:

```
$color-secondary: #DAA520;
```

Then update `index.sass` to include the umuc variables so that they override the variables:

```
@import 'default-variables'
@import 'custom-umuc-variables'

@import 'main'
@import '../components/transcript/competency/competency'
@import '../components/transcript/outcome/outcome'
@import '../components/transcript/unit/unit'
@import '../components/transcript/transcript'
@import 'custom-umuc'
```

And then delete variables.scss.

@pfgray, thoughts?